data/nerdchandise.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "nerdchandise software factory",
+  "type": "personal",
+  "bio": "Software Developers",
```

```suggestion
  "bio": "Software Developer",
```

`Suggestion:` do add some tags, and some more useful links